packages/mobile/src/components/core/FormTextInput.tsx

```diff
@@ -76,6 +79,7 @@ export const FormTextInput = ({
     name,
     validate: required ? validateRequired : undefined
   })
+  const disabled = other.editable === false
```

Comment:
do we want to pass editable down with other, or destructure it just for this component?